src/Combinatorics/Matroids/properties.jl

```diff
@@ -161,7 +161,7 @@ function flats_impl(M::Matroid, r::Union{Int,Nothing}, num_flats::Int, pm_flats)
         jl_flats = reverse(jl_flats)
     end
     matroid_flats = [[M.groundset[i] for i in flat] for flat in jl_flats]
-    if r !== nothing
+    if r!=nothing
```

Comment:
```suggestion
    if r !== nothing
```
Please undo the changes in this file, this should fix the CI errors. (And maybe merge master into your branch)